scripts/migrate_master_azs.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+set -e
+
+installation=$1
+cluster=$2
+
+read -p "Installation: $installation, Cluster: $cluster.
+Press enter to continue, ctrl+c to abort"
+
+opsctl create kubeconfig -i $installation
+gsctl create kubeconfig --cluster=$cluster --certificate-organizations system:masters -e $installation
+
+# get org name
+org_namespace="$(kubectl --context=giantswarm-$installation -n org-giantswarm get cluster -l giantswarm.io/cluster=${cluster} -o yaml -A|yq -r '.items[0].metadata.namespace')"
```

Comment:
In the future when writing these, using JSONPath selector directly with `kubectl` would be better so no additional tooling is required. I don't e.g. personally have `yq` installed. See https://kubernetes.io/docs/reference/kubectl/jsonpath/

No need to rewrite these now since that would be quite a big effort for no gain, but in the future the native approach w/ `kubectl` would be nice :slightly_smiling_face:

Reply:
Can we at least use `jq` with `-o json` ? :sweat_smile:

Reply:
the only reason this PR exists is because Tobi asked for it. It's really meant to be run once by myself only sweat_smile

Reply:
> Can we at least use jq with -o json ? ￼

I guess. I confess that I use that instead of JSONPath w/ adhoc commands. The JSONPath would just look better in scripts as it doesn't strictly require any additional tooling, but then again, the `az` don't have that support so `jq` is pretty mandatory anyway.

Reply:
switched to jq, thanks for the JSONPath hint, will try to remember for next time